CHANGELOG

```diff
@@ -5,10 +5,34 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [1.0.0] 2024-03-04
+
+- This version represents the first public release of the research catalog refactor.
+- This public release will expose the new Homepage and Advanced Search page, the Header and Footer, as well as the Feedback form (SCC-3614).
+- Other pages will fall back to their currently live DFE versions.
+
+### Added
+
+- My Account Header (SCC-3924)
+- Holds API routes (SCC-3953)
```

Comment:
I'd indent lines 17, 18, and 19 under a "My Account" page/feature heading. The Holds API route is specific to the My Account page and there will be a separate "Holds" api route for bibs. So better to differentiate it.